schema.bigquery.json

```diff
@@ -0,0 +1,140 @@
+[
```

Comment:
This file was generated using the following commands

```bash
pip install pyyaml
cargo install jsonschema-transpiler

cat schema.yaml | \
python3 -c 'import json, sys, yaml; y=yaml.safe_load(sys.stdin.read()); print(json.dumps(y))' | \
jq '.schema' | \
jsonschema-transpiler --type bigquery --resolve panic > schema.bigquery.json
```

Reply:
Can you find an appropriate place in the code to add this comment?

Reply:
Added a script `bin/transpile-schema-bigquery.sh`.